test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FrameUpdateCallbackTest.cs

```diff
@@ -620,6 +645,9 @@ private bool OnTouch(object source, View.TouchEventArgs e)
                     SetFrameUpdateCallback(position.X);
                 }
 
+                Vector4 updatedArea = new Vector4(position.X - 10.0f, position.Y - 10.0f, 20.0f, 20.0f);
+                frameUpdateCallback.SetUpdatedArea(updatedArea);
```

Comment:
```suggestion
                // frameUpdateCallback.SetUpdatedArea(updatedArea);
```

I think this code is just for test API, but not be useful for real demo app. Could you please just block this line, and add some comment?